workspace/__lib__/xod-dev/ds-rtc/input-rtc-device/patch.xodp

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
This one is auto-generated and should not be there. @evgenykochetkov @brusherru Do you have an idea how the file was even created on saving?

Reply:
That's my mistake.